### PR TITLE
Change all references to HP-UX package to tar.gz

### PR DESCRIPTION
**Pull request by MiguelazoDS**
## Description

As a consequence of the changes here https://github.com/wazuh/wazuh-packages/pull/2864 to reduce the HP-UX package size, some minor modifications had to be done in the documentation.

image
image

### Docs building
- [x] Compiles without warnings.

### CODE REVIEW

**Review by sebasfalcone — APPROVED** [review]

**Review by sebasfalcone — APPROVED** [review]

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst [hunk 1]
```diff
@@ -10,19 +10,20 @@ The installed agent runs on the endpoint you want to monitor and communicates wi
...
        # groupadd wazuh
        # useradd -G wazuh wazuh
-   
+
 #. Uncompress the package in ``/``.
 
    .. code-block:: console
-   
+
+       # gzip -d wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar.gz
```
Comment (jnasselle) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:26]: Have you tried tar handling the uncompression (tar -xvfz)? IDK if this exists on HP-UX  system, but it does exist on Linux

File: source/upgrade-guide/upgrading-central-components.rst [hunk 2]
```diff
@@ -62,17 +62,21 @@ Upgrading the Wazuh indexer
...
+   curl -X DELETE "https://<WAZUH_INDEXER_IP_ADDRESS>:9200/_index_template/ss4o_*_template" -u <username>:<password> -k
+
+Then, repeat the following steps for every Wazuh indexer node.
 
 #. Disable shard allocation.
 
    .. code-block:: bash
    
-      curl -X PUT "https://<WAZUH_INDEXER_IP_ADDRESS>:9200/_cluster/settings"  -u <username>:<password> -k -H 'Content-Type: application/json' -d'
+      curl -X PUT "https://<WAZUH_INDEXER_IP_ADDRESS>:9200/_cluster/settings" -u <username>:<password> -k -H 'Content-Type: application/json' -d'
```
Comment (jnasselle) [inline source/upgrade-guide/upgrading-central-components.rst:79]: Changes that do not belong to this PR

File: source/upgrade-guide/wazuh-agent/hp-ux.rst [hunk 3]
```diff
@@ -46,6 +46,7 @@ Follow the steps to upgrade the Wazuh agent on HP-UX systems.
 
    .. code-block:: console
 
+      # gzip -d wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar.gz
```
Comment (jnasselle) [inline source/upgrade-guide/wazuh-agent/hp-ux.rst:49]: Ditto

**Review by jnasselle — CHANGES_REQUESTED** [review]

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst (hunk 1 above)
Comment (MiguelazoDS) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:26]: I don't have the VM to take a screenshot now, but it does not recognize the option. I assume the two lines in the script is for the same reason. 
```    
gunzip cmake_3.22.2_HPUX_build.tar.gz
tar -xf cmake_3.22.2_HPUX_build.tar
```

File: source/upgrade-guide/upgrading-central-components.rst (hunk 2 above)
Comment (MiguelazoDS) [reply source/upgrade-guide/upgrading-central-components.rst]: Rebased

File: source/upgrade-guide/wazuh-agent/hp-ux.rst (hunk 3 above)
Comment (MiguelazoDS) [reply source/upgrade-guide/wazuh-agent/hp-ux.rst:49]: https://github.com/wazuh/wazuh-documentation/pull/7107#discussion_r1517912392

**Review by jnasselle — APPROVED** [review]

**Review by sebasfalcone — APPROVED** [review]

**Review by javimed — APPROVED** [review]
